trainingRN/src/screens/todo/layout.js

```diff
@@ -12,8 +13,13 @@ function TodoLayout({ handleDelete, data, handleNew, setNewText }) {
         <View style={styles.taskBox} key={element.key}>
           <Text style={styles.task}>{element.task}</Text>
           <Button title="X" onPress={() => handleDelete(element.id)}/>
+          <CheckBox 
+            checked={element.checked}
+            onPress={() => checkElem(element.id) }
```

Comment:
I'd also extract this function to an instance method

Reply:
Same as above